src/test/java/com/suse/saltstack/netapi/client/SaltStackClientTest.java

```diff
@@ -335,15 +359,44 @@ public void testQueryJobResult() throws Exception {
 
         Map<String, Object> retvals = client.getJobResult("some-job-id");
 
+        verify(1, getRequestedFor(urlEqualTo("/jobs/some-job-id"))
+                .withHeader("Accept", equalTo("application/json")));
+
+
+        Map<String, Map<String, String>> expected =
+                new LinkedHashMap<String, Map<String, String>>(){
+            {
+                put("i3-wm", new LinkedHashMap<String, String>(){
+                    {
+                        put("new", "4.10.1-1");
+                        put("old", "");
+                    }
+                });
+                put("i3lock", new LinkedHashMap<String, String>(){
+                    {
+                        put("new", "2.6-1");
+                        put("old", "");
+                    }
+                });
+                put("i3status", new LinkedHashMap<String, String>(){
```

Comment:
If we keep this collection initialization style, I would personally put a blank between all the `()` and the `{`. Might be only a personal preference though.